content/guides/models/track/runs/multiple-runs-per-process

```diff
@@ -0,0 +1,147 @@
+   - Handy if you want a clear break between sequential sub-runs.
+
+3. **`create_new`**  
+   - Always create a **completely new run**, even if one is already active.  
+   - Does **not** automatically switch the global `wandb.run` to the new run. You must hold onto each run object yourself.  
+   - Ideal for truly concurrent runs (e.g., a “primary” run that remains active while you start/end “secondary” runs).  
+   - **Caution**: Many W&B integrations (e.g., Hugging Face Trainer, Keras callbacks, PyTorch Lightning) assume a single global run. Currently, integrations do not support `create_new` mode.
+
+---
+
+## Example: Maintaining a Primary (Long-Lived) Run + Sub-Runs
+
+Suppose you want to run a **primary task** (like model training) in “Run A,” which stays open for the script’s entire lifespan. Meanwhile, you periodically spawn **short-lived “secondary” runs** (Run B1, B2, etc.) for tasks like evaluation—**without** finishing Run A.
+
+### Workflow Overview
+
+1. **Initialize Run A** and log training metrics.  
+2. **Initialize Run B1** (evaluation), log data, then finish it.  
+3. **Log more data to Run A**.  
+4. **Initialize Run B2**, log data, then finish it.  
+5. **Continue logging to Run A**.  
+6. **Finally finish Run A** at the end.
+
+### Sample Code
+
+```python
+import wandb
+
+def train(name: str) -> None:
+    """
+    Perform one training iteration in its own W&B run.
+
+    Using a 'with wandb.init()' block with `reinit="create_new"` ensures that
+    this training sub-run can be created even if another run (like our primary
+    tracking run) is already active.
+    """
+    with wandb.init(
+        project="my_project",
+        name=name,
+        reinit="create_new"
+    ) as run:
+        # Pretend there's a training loop here logging 'train_loss'.
+        # In a real script, you'd run your training steps inside this block.
+        run.log({"train_loss": 0.42})  # Replace with your real metric(s)
+
+def evaluate_loss_accuracy() -> (float, float):
+    """
+    Returns the current model's loss and accuracy.
+    Replace this placeholder with your real evaluation logic.
+    """
+    return 0.27, 0.91  # Example metric values
+
+# Create a 'primary' run that remains active throughout multiple train/eval steps.
+with wandb.init(
+    project="my_project",
+    name="tracking_run",
+    reinit="create_new"
+) as tracking_run:
+    # 1) Train once under a sub-run named 'training_1'
+    train("training_1")
+    loss, accuracy = evaluate_loss_accuracy()
+    tracking_run.log({"eval_loss": loss, "eval_accuracy": accuracy})
+
+    # 2) Train again under a sub-run named 'training_2'
+    train("training_2")
+    loss, accuracy = evaluate_loss_accuracy()
+    tracking_run.log({"eval_loss": loss, "eval_accuracy": accuracy})
+    
+    # The 'tracking_run' finishes automatically when this 'with' block ends.
+
+---
+
+### Key Takeaways
+
+1. **`reinit="create_new"`**  
+   Forces a new run each time you call `wandb.init()`.
```

Comment:
The word `forces` implies some struggle, like the natural state for `wandb.init()` isn't to create a new run. I would soften this to "Creates a new run ...".